tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "@tsconfig/node16-strictest/tsconfig.json",
   "compilerOptions": {
+    "esModuleInterop": true,
```

Comment:
Why are we re-rewriting this value again? The `@tsconfig/node16-strictest/tsconfig.json`, already has it true.

Reply:
removing